package.json

```diff
@@ -6,14 +6,14 @@
   "author": "webkom",
   "license": "MIT",
   "scripts": {
-    "start": "webpack --config config/webpack.server.js",
+    "start": "webpack --config config/webpack.server.js --mode=development",
     "start:staging": "API_URL=https://lego-staging.abakus.no/api/v1 BASE_URL=https://lego-staging.abakus.no WS_URL=wss://ws-staging.abakus.no CAPTCHA_KEY=6LfS1TQUAAAAAIcxgj4EfkM7lwWeG9efLvS37tCW yarn start",
     "ssr:staging": "API_URL=https://lego-staging.abakus.no/api/v1 BASE_URL=https://lego-staging.abakus.no WS_URL=wss://ws-staging.abakus.no CAPTCHA_KEY=6LfS1TQUAAAAAIcxgj4EfkM7lwWeG9efLvS37tCW yarn ssr",
     "ssr": "echo \"Remember that SSR does not have hot reloading, build with \"yarn build\"\" && node ./dist/server.js",
     "build": "concurrently \"yarn run build:server\" \"yarn run build:client\"",
     "build:all": "concurrently \"yarn build:server\" \"yarn build:client\" \"yarn styleguide:build\"",
-    "build:client": "NODE_ENV=production webpack --config config/webpack.client.js",
-    "build:server": "NODE_ENV=production webpack --config config/webpack.server.js -p",
+    "build:client": "webpack --config config/webpack.client.js --mode=production",
```

Comment:
You've tested this? My understanding was that `--mode=production` was the new way of setting `-p`. Only the `build: server` used to have this flag. Maybe this was intentional(for some reason), as the rational thing would be to build for production.

Reply:
Ahh, yee, it is more or less the same as `-p`, but with a few more optimizations. The new mode has a lot more optimizations that we previously enabled by using `NODE_ENV=production` (or `development`). More info here: <https://webpack.js.org/concepts/mode/>

I have no idea why the client didn't use `-p` before tho.

Reply:
👌